README.md

```diff
@@ -64,6 +64,11 @@ Default volumes
 - `/opt/omero/server/OMERO.server/var`: The OMERO.server `var` directory, including logs
 - `/OMERO`: The OMERO data directory
 
+When using bind mounts, ensure that the ownership and permissions on the host are set as follows:
+
+- host folder mapped to `/opt/omero/server/OMERO.server/var`: UID/GID: 1000/997 `drwxr-xr-x`
+- host folder mapped to `/OMERO`: UID/GID: 1000/0 `drwxr-xr-x`
```

Comment:
@markovendelin, you found it necessary to use GID 0 or is that what docker created? I would have thought 997 would be permitted as well.

NB: unfortunate that the group isn't also 1000 to keep things simple...

Reply:
@markovendelin : sorry for dropping the ball on this one. Since you've said that the UID should be `1000` and the permissions start with `drwx`, the user is going to be able to access regardless of the rest. Perhaps the text is that _either_ UID 1000 _or_ GID 997 should have access?